Mathlib/Probability/Distributions/Gaussian.lean

```diff
@@ -347,6 +348,37 @@ lemma gaussianReal_mul_const {X : Ω → ℝ} (hX : Measure.map X ℙ = gaussian
 
 end Transformations
 
+open Measurable Real
+
+theorem mgf_gaussian {Ω : Type*} {m : MeasurableSpace Ω} {μ : Measure Ω}
+  (X : Ω → ℝ) (hXm : Measurable X) (hX : μ.map X = gaussianReal 0 1) (t : ℝ) :
```

Comment:
Almost everywhere measurability should be enough? That is, replace hXm by `AEMeasurable X μ`.

Reply:
The hypothesis can then be removed entirely, since a.e.-measurability is a consequence of `hX`. Indeed `μ.map X` is defined to be zero if `X` is not a.e.-measurable.

Reply:
I added a supporting lemma `aemeasurable_of_map`. It seems like it should already exist in mathlib but I could not find it. With it, I can easily remove the hypothesis `hXm`